### PR TITLE
[Account 1080] dependencies upgrade

### PR DESCRIPTION
**Pull request by antoinemetifeu**
# Dependencies upgrade

* zoid cannot not updated : The parent & child version should be the same
* sass & sass-loader : I forgot the reason why i don't have updated to the latest versions...
* @vue/cli-plugin-eslint : beta version but watch for next upgrade
* @storybook/vue & @storybook/theming & @storybook/addons & @storybook/addon-docs & @storybook/addon-actions : Updated but already a new version is out (just run npx sb upgrade and try if all build works)
* @storybook/addon-knobs : must be replaced by @storybook/addon-controls

### CODE REVIEW

File: src/components/panel/accountSubComponents/AccountLinkToUi.vue [hunk 1]
```diff
@@ -61,138 +61,138 @@
...
+        employeeId: this.backendUser.employeeId.toString(),
+      }));
+    },
+  },
+  methods: {
+    openLinkShopModal() {
+      this.trackAssociateOrManageAccountButton();
 
-        this.cdcUiDisplayed = true;
-      },
```
Comment (aAmorim27) [inline src/components/panel/accountSubComponents/AccountLinkToUi.vue:179]: We have a regression here, we can't access the shops page from the "View my associated shops" button 
i think that during a merge we lost a part of the function code
You can fix that by adding these lines
```js
openLinkShopModal(action = 'associate') {
      this.action = action;
      this.trackAssociateOrManageAccountButton();

      this.cdcUiDisplayed = true;
    },
``

**Review by aAmorim27 — CHANGES_REQUESTED** [review]
We have a regression when trying to access the associated shops page on a multishop context where some shops aren't linked yet
There is also a problem with the prestakit the background has turned white and some parts of the BO interface are broken
image

**Comment by antoinemetifeu** [comment]
> We have a regression when trying to access the associated shops page on a multishop context where some shops aren't linked yet ~~There is also a problem with the prestakit the background has turned white and some parts of the BO interface are broken~~ image

Hello, can I have more explanation on the problem and how to reproduce it?

**Comment by aAmorim27** [comment]
> > We have a regression when trying to access the associated shops page on a multishop context where some shops aren't linked yet ~There is also a problem with the prestakit the background has turned white and some parts of the BO interface are broken~ image
> 
> Hello, can I have more explanation on the problem and how to reproduce it?

You need to be in a all shops context on a multishop store
- Link the first store but not the others
- Click on the dropdown and click in the "View my associated shops" item
- You will be redirected to the link shop page instead of the shops page

File: src/components/PsAccounts.vue [hunk 2]
```diff
@@ -78,63 +77,68 @@
...
 
-      return {
-        identify,
-        trackAccountComponentViewed,
-      };
+    return {
+      identify,
+      trackAccountComponentViewed,
+    };
+  },
```
[the remark this answers is not in the document]
Comment (antoinemetifeu) [reply src/components/PsAccounts.vue:114]: I will pass all in composition API into ACCOUNT-1105 ticket (the 'context' lib must be a composable to be fully compatible)

**Comment by antoinemetifeu** [comment]
> > > We have a regression when trying to access the associated shops page on a multishop context where some shops aren't linked yet ~There is also a problem with the prestakit the background has turned white and some parts of the BO interface are broken~ image
> > 
> > 
> > Hello, can I have more explanation on the problem and how to reproduce it?
> 
> You need to be in a all shops context on a multishop store
> 
> * Link the first store but not the others
> * Click on the dropdown and click in the "View my associated shops" item
> * You will be redirected to the link shop page instead of the shops page

I will retry but it's certainly fixed by https://github.com/PrestaShopCorp/prestashop_accounts_vue_components/pull/136#discussion_r795538039

**Review by aAmorim27 — APPROVED** [review]

**Comment by antoinemetifeu** [comment]
- Metged into #140

**Pull request closed without merging**